setup.py

```diff
@@ -33,7 +33,7 @@ def build_extensions(self):
 
 def main():
     cpu_count = os.cpu_count() or 8
-    version = "20231030"
+    version = "20231127"
```

Comment:
@rapcmia hey Ralph this change is made by David before STG? let him know that it's already done here